polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -65,6 +66,8 @@ interface IndexTableHeadingBase {
    * When not specified, the value from IndexTable.defaultSortDirection will be used.
    */
   defaultSortDirection?: IndexTableSortDirection;
+  /** Horizontal end spacing around title. Accepts a spacing token. */
+  paddingEnd?: SpaceScale;
```

Comment:
Should we align this prop with the naming of our other [CSS logical props](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) (block/inline)?

```suggestion
  paddingBlockEnd?: SpaceScale;
```